styles/styles.css

```diff
@@ -0,0 +1,139 @@
+/* color1: #09B3C4 */
```

Comment:
You might not use it at ADA but if you use sass (different but similar to css) you can make these constants so you can declare them once and refer to them everywhere! (makes changing the color soooo easy in the future!)